### PR TITLE
doc: fix typo

### PR DESCRIPTION
**Pull request by dmfay**
literally the least I could do 😅 -- I _did_ mean to find and fix a couple other doc issues but couldn't find the `next` docs here or in the graphile.github.io repo

- https://postgraphile.org/postgraphile/next/config#schema-options doesn't mention `defaultBehavior`
- smart-tags link at https://postgraphile.org/postgraphile/next/reserved-keywords/#non-unique-table-names is broken

### CODE REVIEW

**Comment by benjie** [comment]
Thanks! Each root folder contains a website folder for that section, e.g. the PostGraphile website is in `postgraphile/website`. Also the web pages have an “Edit this page” link at the bottom that should take you directly to the right markdown file.

**Comment by dmfay** [comment]
I've added `defaultBehavior`. The broken links seem like a more general problem. Any link to `./target` seems affected -- on /config the screenshots are fine, but "see the [V4 migration docs](https://postgraphile.org/postgraphile/next/config/migrating-from-v4/) for more information" retains a /config/ segment.

**Comment by benjie** [comment]
Ah; that's interesting. It's due to the express static middleware. If you load the URLs via clicking around there's no `/` on the end, but if you reload the page Express redirects from `/postgraphile/next/config` to `/postgraphile/next/config/` and that breaks the links (which are relative). Will have to figure out how to solve that properly in Docusaurus - I expected it would compile out to `/`-anchored links rather than using relative ones!

File: postgraphile/website/postgraphile/config.mdx [hunk 1]
```diff
@@ -482,21 +482,23 @@ to determine the options that they offer.
 
 :::
 
-- `jsonScalarAsString: boolean` - if true, JSON values will be stringified
-  rather than returned as "dynamic" objects.
+- `defaultBehavior: string | undefined` - if set, applies a standard
+  [behavior](./behavior) to introspected objects.
```
Comment (benjie) [inline postgraphile/website/postgraphile/config.mdx:486]:
```suggestion
- `defaultBehavior: string | undefined` - if set, applies a default
  [behavior](./behavior) to all entities; for example to prefer lists
  over connections: `+list -connection`.
```

**Review by benjie — APPROVED** [review]
LGTM; thanks!

**Pull request merged**